### ISSUE TITLE
form field missing a label

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#3906]
A11y issue because form field appears to be missing a label.  This is due to the structure in the HTML where the input tag is included within the label tag:

Example found:
``` python 
src/themes/OLH/templates/elements/journal/article_filter_form.html:
            <label for="section-{{ section.id }}"><input id="section-{{ section.id }}"
                                                         value="{{ section.id }}" type="checkbox"
                                                         name="filter[]"
                                                         {% if section.id in filters %}checked="checked"{% endif %}>{{ section.name }}
            </label>
```

The label tag should be closed before the input tag opens.

**Comment by StephDriver** [comment]
nb nesting input inside label tag is valid HTML, but can cause problems for screenreaders, hence coming up as an A11y issue rather than causing the page not to display properly.
